src/instructlab/__main__.py

```diff
@@ -3,4 +3,4 @@
 # First Party
 from instructlab import lab
 
-lab.cli(None, None)
```

Comment:
I'm guessing this was supposed to be part of the 2nd commit in your series?

Reply:
fixed, the switch from `.cli` to `.ilab` should happen in the early commit but the args change later.